README.md

```diff
@@ -352,12 +352,21 @@ write(backend, table, dfs, condition="", labels=None, max_in_message=0,
 All Frames backends that support the `write` method support the following common parameters:
 
 - <a id="method-write-param-dfs"></a>**dfs** (Required) &mdash; A single DataFrame, a list of DataFrames, or a DataFrames iterator &mdash; One or more DataFrames containing the data to write.
+
+  For the `tsdb` backend, for each DataFrame &mdash;
+
+  - You must define an index column whose value is the sample time of the data.
+    This column serves as the table's primary-key attribute.
+    Note that a TSDB DataFrame cannot have more than one index column of a time data type.
+  - You can optionally define additional string index columns for metric labels.
```

Comment:
1. I would move this to the TSDB specific parameters section
2. Also i would explicitly state that all columns that are not the index are treated as metrics.

Reply:
ACCEPT — I edited the description of the `dfs` param in the common `write` params section to include only the common description and refer to the the `tsdb` backend-specific params for more info, and I added a `dfs` param in the `tsdb` backand params section; I also edited the param description — including to handle item #2 in your comment and your [labels review comment](https://github.com/v3io/frames/pull/306#discussion_r348431617) and I slightly rephrased the description of the opening text in the tsdb backend-specific params section (as it no longer describes only params that are `tsdb` specific). I also made related changes to the internal help text. See commit 8f8fc8e.

Reply:
Approved by Tal